File: DKC/functions/dk_warning.c
#include "dk_warning.h"

//###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ######
#ifndef DKMAIN
#define DKMAIN 1
int main(int argc, char** argv) {
	
	int exit_status;
	exit_status = dk_warning("test dk_warning() \n");
	exit_status = dk_warning("test dk_warning() with number: %d\n", WARNING);
	exit_status = dk_warning("test dk_warning() with string: %s\n", "WARNING string");
	return exit_status;
}
#endif

File: DKPhp/functions/dk_warning.php
#!/usr/bin/php
<?php

################################################################################
# dk_warning(message)
#
#    Print a warning message to the console
#
#    @message - The message to print
#
function dk_warning() {
	#dk_debugFunc(1);
	dk_source("dk_log");
	
	if(!isset($argv)){
		$argv = func_get_args();
	}
	
    dk_log("WARNING", $argv[0]);
}




###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ######
if(!function_exists('DKTEST')){ function DKTEST() {
	#dk_debugFunc 0

    dk_warning("test dk_warning message");
}}


include_once(str_replace("\\","/",$_SERVER['USERPROFILE'])."/Digital Knob/Development/DKPhp/functions/DK.php");
?>